src/libs/ReportActionsUtils.js

```diff
@@ -56,6 +56,14 @@ function isMoneyRequestAction(reportAction) {
     return lodashGet(reportAction, 'actionName', '') === CONST.REPORT.ACTIONS.TYPE.IOU;
 }
 
+/**
+ * @param {Object} reportAction
+ * @returns {Boolean}
+ */
+function isReportPreviewAction(reportAction) {
+    return lodashGet(reportAction, 'actionName', '') === CONST.REPORT.ACTIONS.TYPE.REPORTPREVIEW;
```

Comment:
Maybe I am crazy, but does anyone recall discussion about replacing lodashGet useages with an alternative?

Reply:
I am not sure

Reply:
Okay cool, I'll resolve then